src/js/config.js

```diff
@@ -22,13 +22,15 @@ export const CSS_SELECTOR_REPUBLISHING_HEADER_LINK = '.o-header-services__nav-li
 export const CSS_SELECTOR_REPUBLISHING_BTN = `${CSS_SELECTOR_CONTENT_ID}[${ATTR_SYNDICATED}="true"].${CSS_CLASS_REPUBLISHING_BUTTON}`;
 export const CSS_SELECTOR_STORY_LINK = '.story__link'; // for top fastFT bar under main navigation
 export const CSS_SELECTOR_SYNDATION_ICON = `${CSS_SELECTOR_CONTENT_ID}[${ATTR_SYNDICATED}="true"].${CSS_CLASS_PREFIX}-icon`;
+export const CSS_SELECTOR_TOPIC_CARD_LINK = 'a.topic-card__concept-article-link';
```

Comment:
not this specifically - wouldn't this all be easier to manage if we used a predefined data attribute? e.g. data-ui="syndication"

Reply:
True. There is that kind of thing here (`data-syndicatable-uuid`), but not sure if/how it's used: https://github.com/Financial-Times/n-syndication/blob/master/src/js/index.js#L63-L70

Reply:
That would definitely be an improvement for sure @debugwand I feel like this way is becoming a little bit unwieldy